spec.bs

```diff
@@ -3987,6 +3993,36 @@ in the [=user agent=]'s [=interest group set=] and whose [=map/values=] are [=k-
 This allows the browser to rerun portions of an auction without incurring the delay (and added side channels)
 from querying the server during an auction.
 
+<div algorithm>
+  To <dfn>create a k-anon restricted deep copy of the interest group</dfn> given an [=interest group=] |ig|:
+    1. Let |kAnonRestrictedIG| be a deep copy of |ig|
+    1. If |ig|'s [=interest group/ads=] is not null:
+      1. Set |kAnonRestrictedIG|'s [=interest group/ads=] to an empty [=list=] of [=interest group ad=].
+      1. [=list/For each=] |igAd| of |ig|'s [=interest group/ads=]:
+        1. Let |adHashCode| be the result of [=compute the key hash of ad=] given |ig| and |igAd|.
+        1. If [=query k-anonymity cache=] for |adHashCode| returns true:
+          1. If |igAd|'s [=interest group ad/selectable buyer and seller reporting IDs=] is not null:
+          1. Let |kAnonRestrictedSelectableReportingIds| be a new empty [=list=] of [=string=]s.

```

Comment:
the indention here is wrong. Which steps below should be under this If block?

Reply:
fixed it! thanks!